nautobot/users/api/serializers.py

```diff
@@ -66,6 +67,26 @@ class Meta:
         fields = ("id", "url", "name", "user_count")
 
 
+class TokenSerializer(ValidatedModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="users-api:token-detail")
+    key = serializers.CharField(min_length=40, max_length=40, allow_blank=True, required=False)
+    user = NestedUserSerializer()
+
+    class Meta:
+        model = Token
+        fields = ("id", "url", "display", "user", "created", "expires", "key", "write_enabled", "description")
```

Comment:
```suggestion
    class Meta:
        model = Token
        fields = ("id", "url", "display", "created", "expires", "key", "write_enabled", "description")
```

User is always self now.